R/SeqComparisonTable.R

```diff
@@ -0,0 +1,151 @@
+# SeqComparisonTable.R
+#
+# Purpose: Exploratory System Analysis tool project for BCB420 2019
```

Comment:
The "general" function/script header is actually not needed when writing a roxygen header. Information should be merged there.